### DISCUSSION TITLE
getting_started/introduction/first_look_at_the_editor

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#66] [User-contributed notes]
# getting_started/introduction/first_look_at_the_editor

This page will give you a brief overview of Godot's interface. We're going to look at the different main screens and docks to help you situate yourself. The Project Manager: When you launch Godot, ...

https://docs.godotengine.org/en/stable/getting_started/introduction/first_look_at_the_editor.html

**Comment by CyberFoxar** [comment]
One last thing I'd add.. Maybe not here, but somewhere else, is to highlight the settings, somewhere. Especially the `Editors>3d>Navigation Scheme...` dropdown, which has stumped us for a whille when onboarding a Unity person. 

They were used to Unity's scheme (which correspond to the `maya` scheme), and it took us a while to find out... especially since the few resources there are for people switching over mostly gloss over the feel. This might be a wortwhile addition to one of the pages in this "Getting Started".

**Reply by vgezer** [reply]
Maybe this: [Navigating the 3D environment](https://docs.godotengine.org/en/stable/tutorials/3d/introduction_to_3d.html#navigating-the-3d-environment)
